src/components/Router.js

```diff
@@ -84,13 +76,19 @@ class Router extends Component {
               <Route
                 path="/login-redirect"
                 render={() => <LoginRedirect />} />
+              <Route
+                path="/forbidden"
+                render={() => <Forbidden />} />
 
+              <AdminRoute 
```

Comment:
Does this need to be an admin route? 
If they are an admin they won't have a student_id so it will show NewMemberRegister anyways

Reply:
Gooddd point!